src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --regular-font: #000;
+  --hover-font: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 14px;
```

Comment:
`line-height` is better to set with relative value or in percentage.
For example:
```
line-height: 1;
line-height: 1.5;
line-height: 150%;
```
I like the relative values better.